content/applications/hr/employees/new_employee.rst

```diff
@@ -157,6 +169,10 @@ To delete any line from the :guilabel:`Resumé` tab, click the :guilabel:`🗑
 delete the entry. Add a new line by clicking the :guilabel:`Add` button next to the corresponding
 section.
 
+.. important::
+   Only user's with :guilabel:`Officer: Manage all employees` or :guilabel:`Administrator` rights
+   for the *Employees* app can add or edit skills.
+
 .. _employees/skill-types:
 
 Skill Types
```

Comment:
Thanks for the suggestion! I think I will leave it here, only because I do like to go in the order of the steps as I would do it/in a logical order on the form. I understand your logic, too, but since I am using this specific 'roadmap' if you will (top info, tab info, each tab explained in detail) I will leave this here. I did add another custom anchor, and now that the info has been updated in the skills section I think it makes more sense. So when someone is doing a new employee form, and they do have the skill types all set, this is a much better flow and makes more sense, as they can skip that part.